src/styles/sidebar/components/LaunchErrorPanel.scss

```diff
@@ -1,8 +1,6 @@
 .LaunchErrorPanel {
   // Top margin ensures that the error notice appears below the Notebook's
   // "Close" button.
-  margin-top: 50px;
-
-  margin-left: 5px;
-  margin-right: 5px;
+  margin: 0.5rem;
+  margin-top: 3rem;
```

Comment:
The net effect here is nearly identical, just translated to `rem` as that's the new way of our world.